ray-on-gke/tpu/kuberay-tpu-webhook/webhook_main_test.go

```diff
@@ -1126,33 +953,31 @@ func Test_CheckWorkersMatchTopology(t *testing.T) {
 }
 
 func Test_ValidateRayCluster(t *testing.T) {
-	setupTest(t)
-
 	tests := map[string]struct {
-		rayCluster       	*rayv1.RayCluster
-		topology         	string
-		numOfHosts       	int32
-		replicas         	*int32
-		missingWorkerGroups	bool
-		expectedResponse 	*admissionv1.AdmissionResponse
-		expectedAllowed  	bool
-		expectedResult   	*metav1.Status
+		rayCluster          *rayv1.RayCluster
+		topology            string
+		numOfHosts          int32
+		replicas            *int32
+		missingWorkerGroups bool
+		expectedResponse    *admissionv1.AdmissionResponse
+		expectedAllowed     bool
+		expectedResult      *metav1.Status
 	}{
 		"validateRayCluster no workerGroupSpecs": {
 			// doesn't create any workergroups, pass-through
-			rayCluster:      	 testRayCluster.DeepCopy(),
-			topology:        	 "",
-			numOfHosts:      	 int32(1),
+			rayCluster:          getTestRayCluster("test-cluster", "test-group", "test-namespace", int32(1), 1),
+			topology:            "",
+			numOfHosts:          int32(1),
 			missingWorkerGroups: false,
-			expectedAllowed: 	 true,
-			expectedResult: 	 &metav1.Status{
-									Status:  "Success",
-									Message: "",
-								},
+			expectedAllowed:     true,
+			expectedResult: &metav1.Status{
+				Status:  "Success",
+				Message: "",
+			},
 		},
 		"validateRayCluster no TPUs requested": {
 			// doesn't request TPUs, pass-through
-			rayCluster:      testRayCluster.DeepCopy(),
+			rayCluster:      getTestRayCluster("test-cluster", "test-group", "test-namespace", int32(1), 1),
```

Comment:
I think this rayCluster is still requesting TPUs?

Reply:
Fixed in 6614355